index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css"/>
+</head>
+<body>
+    <div class="menu_bar">
+        <div class="logo">My Logo</div>
+        <div class="menu_icon"><img src="images/Union.png"/></div>        
+    </div>
+    <div class="intro">
+        <div class="headline">
+            <main>
+                <h2>
```

Comment:
- [ ] I kindly suggest that you make only your main title of "I’m May Pyone. Glad to see you" an `h1`. This will enable google (and other search engines) knows about the main title of the page, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.